hw/ip/sram_ctrl/lint/sram_ctrl.waiver

```diff
@@ -3,3 +3,7 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 # waiver file for SRAM controller
+
+# All leaf resets have a reset multiplex
+waive -rules PARAM_NOT_USED -location {sram_ctrl.sv} -regexp {.*RndCnstSramLfsrPerm.*} \
```

Comment:
@msfschaffner

Reply:
Rather than have to add this separately to the Verilator file as well, I think you might be able to define an `unused_foo` signal in the RTL to handle both tools. Maybe cleaner?

Reply:
o you mean define an RTL signal to take on the value of the parameter?
I did not think about that.  Good suggestion.